Bitbucket.Authentication/Src/AuthenticationResult.cs

```diff
@@ -116,9 +116,9 @@ public AuthenticationResult(AuthenticationResultType type, Token token, string u
     /// </summary>
     public enum AuthenticationResultType
```

Comment:
Odd. Am I misunderstanding something about GH PRs? This is already changed in master, I rebased on top of it, yet the diff still shows here?